static/app/views/starfish/views/spans/spanSummaryPanel/useApplicationMetrics.tsx

```diff
@@ -1,17 +1,26 @@
 import {useQuery} from 'sentry/utils/queryClient';
+import usePageFilters from 'sentry/utils/usePageFilters';
+import {getDateQueryFilter} from 'sentry/views/starfish/modules/databaseModule/queries';
 import {HOST} from 'sentry/views/starfish/utils/constants';
+import {getDateFilters} from 'sentry/views/starfish/utils/dates';
 
 type Metrics = {
   count: number;
   total_time: number;
 };
 
 export const useApplicationMetrics = (referrer = 'application-metrics') => {
+  const pageFilters = usePageFilters();
+  const {startTime, endTime} = getDateFilters(pageFilters);
+  const dateFilters = getDateQueryFilter(startTime, endTime);
+
   const query = `
   SELECT
   count() as count,
   sum(exclusive_time) as total_time
   FROM spans_experimental_starfish
+  WHERE 1 = 1
```

Comment:
```suggestion
```

Reply:
acc this might be needed cause i think the date filter prepends and `AND`

Reply:
I think this is here so that we can tack on the additional AND clause with the date filter, in case it's undefined

Reply:
Yup, that's right! Gross hack, but does the trick